test/models/movie_test.rb

```diff
@@ -0,0 +1,16 @@
+require "test_helper"
+
+class MovieTest < ActiveSupport::TestCase
+  test "should pass all validates" do
```

Comment:
Vi er ret glade for at bruge minitest spec til at skrive tests med. Så kan man skrive:

```
describe "the fancy method" do
  it "deletes facebook" do
  end

  it "also formats your harddrive" do
  end
end
```

En af grundene til at det lidt rart at have med, er at det hjælper en til at tænke lidt over test beskrivelsen **it** does-something-and-i-want-to-test-that **do**

Så det kan du fint lige afprøve :)